galaxy/values.yaml

```diff
@@ -75,8 +75,9 @@ gpsql:
     pullPolicy: IfNotPresent
   postgresqlDatabase: galaxy
   postgresqlPassword: ""
-  postgresqlDataDir: /var/lib/postgresql/data
+  postgresqlDataDir: "/var/lib/postgresql/data"
   persistence:
     enabled: true
     existingClaim: postgres-pv-claim
     storageClass: manual
+    mountPath: "/var/lib/postgresql"
```

Comment:
Why do we need this? The chart docs state this is the same as `postgresqlDataDir`?

Reply:
I think we need it because we are using the docker image: postgresDataDir must be a subpath of mountPath there: https://github.com/helm/charts/tree/master/stable/postgresql#deploy-chart-using-docker-official-postgresql-image